src/openforms/forms/tests/admin/test_views.py

```diff
@@ -305,3 +308,271 @@ def test_single_import(self, m_import):
             "Form successfully imported!",
             messages[0].message,
         )
+
+
+@disable_admin_mfa()
+class TestMigrationForm(WebTest):
+    csrf_checks = False
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        cls.admin_user = StaffUserFactory(user_permissions=["forms.change_form"])
```

Comment:
```suggestion
        cls.admin_user = StaffUserFactory.create(user_permissions=["forms.change_form"])
```
*always* use `Factory.build` or `Factory.create` explicitly and don't rely on the implicit create behaviour.